### PR TITLE
fixed container aware command depreciation

### PR DESCRIPTION
**Pull request by N-M**
Fixed depreciations for commands. Tests fix is needed.

### CODE REVIEW

**Comment by wgcvc** [comment]
Hello @N-M 

Is this still a work in progress? 
do you need help with the failed tests?

thanks

**Comment by N-M** [comment]
Hi @wgcvc, 

I have pushed the latest fix. It would be great if you could help with failed tests.

**Comment by trandangtri** [comment]
Hallo @N-M and @wgcvc, any new update for this issue? I'm still here if you guys need any helps or explanation.

**Comment by N-M** [comment]
Hi @trandangtri, I need help in setting up container in phpunit. Other then that this pull request should be fine.

**Comment by wcomnisky** [comment]
I've opened a PR setting up the container for the tests:
https://github.com/N-M/sqs-queue-bundle/pull/2

**Comment by wcomnisky** [comment]
How to fix the _continuous-integration/styleci/pr_ ? What should we check to meet its requirements/conditions? Couldn't see something useful on its Details.

**Comment by N-M** [comment]
That is a code style check. I can fix that.

**Comment by wcomnisky** [comment]
Voilà! 👏

File: Command/QueueCreateCommand.php [hunk 1]
```diff
@@ -73,27 +76,27 @@ protected function configure()
...
-            'MaximumMessageSize' => $input->getOption('maximum_message_size'),
-            'MessageRetentionPeriod' => $input->getOption('message_retention_period'),
+            'DelaySeconds'                  => $input->getOption('delay_seconds'),
+            'MaximumMessageSize'            => $input->getOption('maximum_message_size'),
+            'MessageRetentionPeriod'        => $input->getOption('message_retention_period'),
             'ReceiveMessageWaitTimeSeconds' => $input->getOption('receive_message_wait_time_seconds'),
-            'VisibilityTimeout' => $input->getOption('visibility_timeout'),
-            'ContentBasedDeduplication' => $input->getOption('content_based_deduplication')
+            'VisibilityTimeout'             => $input->getOption('visibility_timeout'),
+            'ContentBasedDeduplication'     => $input->getOption('content_based_deduplication'),
```
Comment (trandangtri) [inline Command/QueueCreateCommand.php]: Dear @N-M ,
Highly appreciated your work. Just a tiny request, would you please disable aligning key-value pairs in your IDE for consistent purpose? Then I could merge. Thanks in advance

Comment (N-M) [reply Command/QueueCreateCommand.php]: Sure I will do it. Reason why I aligned them is because that was the error that we got from styleci. Thanks

Comment (trandangtri) [reply Command/QueueCreateCommand.php]: Thanks for your contribution!

**Pull request merged**